.github/workflows/release.yml

```diff
@@ -77,6 +77,8 @@ jobs:
               done
             fi
             ./tools/release.sh
+            ./tools/packaging/generate_debian_package.sh aarch64 build-opt/dragonfly-aarch64
```

Comment:
@boazsade  no need to pass architecture argument anymore

Reply:
you're right!